declarative/declarativemprisplayer.cpp

```diff
@@ -198,6 +204,41 @@ using namespace Amber;
     Defaults to false.
 */
 
+/*!
+    \qmlproperty bool MprisPlayer::canShuffle
+    \brief Indicates whether the player can control the shuffle status
+
+    Should be set to true if the shuffle status of the play queue can be
+    controlled. If set to true, remote controllers can set the shuffle
```

Comment:
To be exact this is not just whether you can control shuffle but whether you even see the value of it.

Reply:
I adjusted this.